mfhdf/libsrc/putget.c

```diff
@@ -430,44 +425,14 @@ xdr_NCvbyte(XDR *xdrs, unsigned rem, unsigned count, char *values)
          * We will read in the word to change one byte in it.
          */
         origin = xdr_getpos(xdrs);
-#ifdef XDRSTDIO
-        /*
-         * N.B. : "a file positioning function must be called between
-         * a write and a read or vice versa"
-         *        - limitations of stdio, open for update
-         */
-        if (!xdr_setpos(xdrs, origin))
-            return (FALSE);
-#endif /* XDRSTDIO */
-        /* next op is a get */
+
```

Comment:
I don't know what the deal is about using xdrstdio.c instead of xdrposix.c, but is that the right thing to do in this PR?

Reply:
I seriously doubt this is used by anyone, but we can discuss